020/main.go

```diff
@@ -0,0 +1,24 @@
+// Daily Joel #20
+// Methods.
+package main
+
+import "fmt"
+
+type Person struct {
+	name string
+}
+
+func (p *Person) setName(name string) {
+	p.name = name
+}
```

Comment:
A method with a pointer receiver.

Reply:
And this is the difference with 20, that we're using a method. 
If we want to modify the type Person using this method we need a pointer